src/containers/ListView/index.jsx

```diff
@@ -92,6 +88,26 @@ export class ListView extends React.Component {
               Header: 'Grading Status',
               accessor: 'gradeStatus',
               Cell: this.formatStatus,
+              Filter: DropdownFilter,
+              filter: 'equals',
+              filterChoices: [
+                {
+                  name: 'Ungraded',
+                  value: 'ungraded',
+                },
+                {
+                  name: 'Locked',
+                  value: 'locked',
+                },
+                {
+                  name: 'Graded',
+                  value: 'graded',
+                },
+                {
+                  name: 'In Progress',
+                  value: 'in-progress'
+                },
+              ]
```

Comment:
This was for a separate ticket, and should probably be in its own PR

Reply:
It is not polish to be honest. It was more of a discovery. I just wanted to get ride of the change was in my stash 😅 . I don't mind remove it for now.